marc.go

```diff
@@ -313,3 +317,49 @@ func contentType(x byte) string {
 	}
 	return t
 }
+
+// RetrieveLanguageCodelist retrieves language codes for parsing MARC languages
+func RetrieveLanguageCodelist() (map[string]string, error) {
+	file, err := os.Open("fixtures/languages.xml")
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Language struct
+	type Language struct {
+		Name string `xml:"name"`
+		Code string `xml:"code"`
+	}
+
+	decoder := xml.NewDecoder(file)
+	languages := make(map[string]string)
+
+	for {
+		t, _ := decoder.Token()
+		if t == nil {
+			break
+		}
+		switch se := t.(type) {
+		case xml.StartElement:
+			if se.Name.Local == "language" {
+				var l Language
+				decoder.DecodeElement(&l, &se)
+				languages[l.Code] = l.Name
+			}
+		}
+	}
+	return languages, err
+}
+
+// TranslateLanguageCodes takes an array of MARC language codes and returns the language names.
+func TranslateLanguageCodes(x []string, languageCodes map[string]string) []string {
```

Comment:
`x` is probably not the best name for this variable ;)

Reply:
👍 will fix, residual from copy/paste :)